src/factory/universal-ad-id.js

```diff
@@ -0,0 +1,9 @@
+export default (universalAdId, $universalAdId) => {
+  if ($universalAdId.idRegistry) {
```

Comment:
We seem to be mixing `if (object)` and `if (object != null)`. It's not a huge issue so I don't want it to block this PR, but let's try to make sure that everything inside `if (...)` evaluates to a boolean in the future.